Sources/SQLite/Database/SQLiteConnection.swift

```diff
@@ -36,32 +35,35 @@ public final class SQLiteConnection: BasicWorker, DatabaseConnection, DatabaseQu
 
     /// Reference to parent `SQLiteDatabase` that created this connection.
     /// This reference will ensure the DB stays alive since this connection uses
-    /// it's C pointer handle.
-    internal let database: SQLiteDatabase
+    /// it's thread pool.
+    private let database: SQLiteDatabase
+
+    /// Internal SQLite database handle.
+    internal var handle: OpaquePointer!
```

Comment:
@tanner0101 What do you think about using an IUO for storing the handle. This way `isClosed` simply becomes a computed property.

Reply:
Could we use a regular optional instead? I'd prefer to force unwrap the optional as needed over have that behavior be implicit.

Reply:
I just tried. The C calls do accept normal optionals, so the compiler won't tell you to force unwrap it.